core/cap-0023.md

```diff
@@ -344,6 +341,20 @@ The behavior of `CreateClaimableBalanceOp` is as follows:
     - `asset` as specified in the operation
     - `amount` as specified in the operation
     - `reserve` equal to `claimants.size() * baseReserve`
+
+[^id-arithmatic]: ```
+    OperationID = (
+      sourceAccount ||
+      seqNum ||
+      opNum
+    )
+    balanceID.v0() = hex(sha256(OperationID))
```

Comment:
I would remove `hex()` from here. The protocol just uses the xdr `Hash` type. Someone may look at this and think it's a hexadecimal string.

Reply:
The `Hash` type application wasn't immediately apparent to me. What do you think about a line explicitly defining the final protocol definition separate from a client ID? I think mentioning the hash in the CAP is worthwhile since it's what ends up in [production applications](https://github.com/blocktransfer/py-tad3-horizon/blob/a9be280b365ee1eb74eea51376c42958c1b516bc/investors/stockSplit.py#L135).